include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -231,6 +231,29 @@ namespace parallel
       virtual bool
       is_multilevel_hierarchy_constructed() const override;
 
+      /**
+       * Save the triangulation into the given file. This file needs to be
+       * reachable from all nodes in the computation on a shared network file
+       * system. See the SolutionTransfer class on how to store solution vectors
+       * into this file. Additional cell-based data can be saved using
+       * register_data_attach().
```

Comment:
```suggestion
       * DistributedTriangulationBase::register_data_attach().
```
I am not sure whether `doxygen` will correctly reference inherited functions from the base class without specifying the class/namespace here and in other occasions. could you verify it by building the documentation locally?